androminion/res/values-de/strings-ui.xml

```diff
@@ -168,6 +181,7 @@
 	<string name="played_header">Gespielt:</string>
 	<string name="tavern_header">Wirtshaus:</string>
 	<string name="archive_header">Archiv:</string>
+    <string name="crypt_header">Gruft:</string>
```

Comment:
Official translation for Crypt is Krypta

Reply:
where do you find the official translation?

Reply:
I looked at the translation for Dominion Online.